docs/index.md

```diff
@@ -66,32 +26,196 @@ import {Query, getMonths, getYears, getCategories} from "./components/queries.js
 // misc
 import * as echarts from "npm:echarts";
 import {svg} from "npm:htl";
+import * as d3 from "npm:d3";
 ```
 
 ```js
 // load data
 const crimeData = await loadCrimeData();
 const geoData = await loadGeometryData();
+geoData.features.forEach((g) => {
+    // coordinates are inside out, so reverse them
```

Comment:
Maybe save this already reversed in the json file? Do we ever use the normal coordinates?

Reply:
I don't think we do, but it's really not computationally intensive. there are only 25 regions